classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    students << student
```

Comment:
- [ Required ] I think the `student` object being passed into the `add_student` method should be pushed into the instance variable `@students`. Currently, it's not being pushed into the `@students` instance variable but into a local variable `students` . Please refactor your code as shown below.

```ruby
  def add_student(student)
    @students << student
    student.classroom = self
  end
```